crates/turbo-tasks-build/src/lib.rs

```diff
@@ -162,7 +162,6 @@ pub fn rerun_if_glob(globs: &str, root: &str) {
     let cwd = env::current_dir().unwrap();
     let globs = cwd.join(globs.replace('/', PATH_SEP.to_string().as_str()));
     let root = cwd.join(root.replace('/', PATH_SEP.to_string().as_str()));
-    println!("cargo:rerun-if-changed={}", root.display());
```

Comment:
This is important to detect a new directory added

Reply:
But then we only need this and not the others because it's recursive according to the docs?